packages/react/src/runtimes/external-store/createMessageConverter.tsx

```
@@ -0,0 +1,55 @@
+    toOriginalMessages: (
+      input: ThreadState | ThreadMessage | ThreadMessage["content"][number],
+    ) => {
+      const messages = getExternalStoreMessages(input);
+      if (messages.length === 0) throw new Error("No original messages found");
+      return messages;
+    },
+    toOriginalMessage: (
+      input: ThreadState | ThreadMessage | ThreadMessage["content"][number],
+    ) => {
+      const messages = result.toOriginalMessages(input);
+      return messages[0]!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)